go.mod

```diff
@@ -118,6 +117,7 @@ require (
 	github.com/mattn/go-isatty v0.0.20 // indirect
 	github.com/ncruces/go-strftime v0.1.9 // indirect
 	github.com/pjbgf/sha1cd v0.3.0 // indirect
+	github.com/pkg/errors v0.9.1 // indirect
```

Comment:
Can you check which transitive dep uses this? If it's `argo/pkg` for instance, we could refactor it out of there as well

Reply:
Yes it's argo/pkg which looks like it hasn't been updated in a long time.

Reply:
We do infrequently update code there when we need to support features on Workflows (e.g. some expression templating code and some S3 code).

Do you think you could take a look and do a refactor there too? That way all Argo projects can stop depending on it indirectly

Reply:
Completed in https://github.com/argoproj/pkg/pull/747